.github/workflows/pod_failover_test.yml

```diff
@@ -0,0 +1,118 @@
+name: Pod Failover Test
+on:
+    schedule:
+    - cron: "0 13 * * *"
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      AZURE_TENANT_ID: ${{ secrets.TenantId }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.SubscriptionId }}
+      AZURE_CLIENT_ID: ${{ secrets.ClientId }}
+      AZURE_CLIENT_SECRET: ${{ secrets.ClientSecret }}
+      AZURE_LOCATION: ${{ secrets.E2ELocation }}
```

Comment:
We're hitting quota limits in the default E2E region. We should probably just hardcode this to northeurope where we have more quota capacity.

Reply:
Changed to northeurope